plugins/woocommerce-admin/client/marketplace/components/content/content.tsx

```diff
@@ -104,11 +105,29 @@ export default function Content(): JSX.Element {
 	const renderContent = ( tab: string, products: Product[] ): JSX.Element => {
 		switch ( tab ) {
 			case 'extensions':
-				return <Extensions products={ products } />;
+				return (
+					<Extensions
+						products={ products }
+						categorySelector={ true }
+						label={ 'extension' }
+						labelPlural={ 'extensions' }
+					/>
+				);
 			case 'themes':
-				return <Themes products={ products } />;
+				return (
+					<Extensions
+						products={ products }
+						categorySelector={ true }
+						label={ 'theme' }
+						labelPlural={ 'themes' }
+					/>
+				);
 			case 'search':
 				return <SearchResults products={ products } />;
+			case 'search-extensions':
+				return <SearchExtensions products={ products } />;
+			case 'search-themes':
+				return <SearchThemes products={ products } />;
```

Comment:
We could use `SearchResults` with a `type` property. The current implementation adds extra components that don't do a lot and could easily be merged into `SearchResults`.

Reply:
Done in 6263b0025b807d0d3ea3089b0cb774bad59f61d2